doc/modules/linear_model.rst

```diff
@@ -1051,6 +1057,62 @@ considering only a random subset of all possible combinations.
 
     .. [#f2] T. Kärkkäinen and S. Äyrämö: `On Computation of Spatial Median for Robust Data Mining. <http://users.jyu.fi/~samiayr/pdf/ayramo_eurogen05.pdf>`_
 
+.. _huber_regression:
+
+Huber Regression
+----------------
+
+The :class:`HuberRegressor` is an extension of :class:`Ridge` except that
```

Comment:
I wouldn't say it is an extension. Maybe say it is similar in that it also applies L2 regularization and a squared loss for "inliers"?